realm/realm-library/src/main/java/io/realm/RealmQuery.java

```diff
@@ -1248,6 +1248,36 @@ public boolean isValid() {
         return this;
     }
 
+    // Like
+
+    /**
+     * Condition that the value of field matches with the specified substring, with wildcards.
+     *
+     * @param fieldName the field to compare.
+     * @param value the wildcard string.
+     * @return the query object.
+     * @throws java.lang.IllegalArgumentException One or more arguments do not match class or field type.
```

Comment:
`One or more` -> `if one or more`. Same for the following. I think there was a doc to follow, but i cannot find it ... maybe this one? http://blog.joda.org/2012/11/javadoc-coding-standards.html